applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -2,38 +2,41 @@
 import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
 import math
 
-
 def Factory(settings, Model):
     if(not isinstance(settings, KratosMultiphysics.Parameters)):
         raise Exception(
             "expected input shall be a Parameters object, encapsulating a json string")
 
-    return DefineWakeProcess(Model, settings["Parameters"])
-
+    return DefineWakeProcess2D(Model, settings["Parameters"])
 
-class DefineWakeProcess(KratosMultiphysics.Process):
+# TODO Implement this process in C++ and make it open mp parallel to save time selecting the wake elements
+class DefineWakeProcess2D(KratosMultiphysics.Process):
     def __init__(self, Model, settings):
+        # Call the base Kratos process constructor
         KratosMultiphysics.Process.__init__(self)
 
-        default_settings = KratosMultiphysics.Parameters("""
-            {
-                "mesh_id"                   : 0,
-                "model_part_name"           : "please specify the model part that contains the kutta nodes",
-                "upper_surface_model_part_name" : "please specify the model part that contains the upper surface nodes",
-                "lower_surface_model_part_name" : "please specify the model part that contains the lower surface nodes",
-                "fluid_part_name"           : "MainModelPart",
-                "wake_direction"                 : [1.0,0.0,0.0],
-                "epsilon"    : 1e-9
-            }
-            """)
-
+        # Check default settings
+        default_settings = KratosMultiphysics.Parameters(r'''{
+            "model_part_name": "",
+            "wake_direction": [1.0,0.0,0.0],
```

Comment:
The wake direction is actually always the direction of the free stream velocity. I say this because in the GiD interface one needs to specify both and only one is necessary

Reply:
> Does this mean that we can remove the `wake_direction`?
> 
> Regarding the `epsilon`, I saw that you removed it in the GUI. Do you want to keep it in here as default value?

Yes, we can keep it as default here.